ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/FontProvider.kt

```diff
@@ -0,0 +1,16 @@
+package com.revenuecat.purchases.ui.revenuecatui.fonts
+
+import androidx.compose.ui.text.font.FontFamily
+
+/**
+ * Implement this interface to provide custom fonts to the PaywallView. If you don't, the current material3 theme
+ * typography will be used.
+ */
+interface FontProvider {
+    /**
+     * Returns the font to be used for the given [TypographyType]. If null is returned, the default font will be used.
+     * @param type the [TypographyType] for which the font is being requested.
+     * @return the `FontFamily` to be used for the given [TypographyType].
+     */
+    fun getFont(type: TypographyType): FontFamily?
```

Comment:
Unfortunately, we can't use this for the PaywallActivity wrapper, since `FontFamily` is not parcelable. This will only work when launching the paywall through compose.

Reply:
Maybe add a comment mentioning that, and referencing `FontResourceProvider`?